CMakeLists.txt

```diff
@@ -167,9 +167,9 @@ include(add_enclave)
 # User configurable options
 # DCAP library support on Windows is experimental and is disabled by default
 if (UNIX)
-  option(USE_LIBSGX "Build oehost using SGX library requiring FLC" ON)
+  option(HAS_QUOTE_PROVIDER "Get/verify quotes using DCAP requiring FLC" ON)
 elseif (WIN32)
-  option(USE_LIBSGX "Build oehost using SGX library requiring FLC" OFF)
+  option(HAS_QUOTE_PROVIDER "Get/verify quotes using DCAP requiring FLC" OFF)
```

Comment:
It's because the comment above it: `# DCAP library support on Windows is experimental and is disabled by default`. 

Are you saying this statement is not true anymore?

Reply:
Yes, the goal of the v0.7 release was to officially add DCAP support to Windows. @radhikaj for her opinion on this.

Reply:
@CodeMonkeyLeet I agree. This is an outdated comment. Thanks for pointing it out @jxyang "DCAP library support on Windows is experimental and is disabled by default."

Reply:
Removed asem related code. 

Turned on HAS_QUOTE_PROVIDER for both Linux and Windows by default.